components/story/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import ReactMarkdown from 'react-markdown';
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import readme from '../README.md';
+
+storiesOf( 'Components', module )
+	.add( 'Welcome', () => <ReactMarkdown source={ readme } /> );
```

Comment:
Do we want to add `withKnobs` as a decorator here? Currently when navigating from Button to Welcome, the Button's knobs remain.